### DISCUSSION TITLE
Open a user's chat in a new tab/split from usercard

### DISCUSSION THREAD

**Discussion by badoge** [Chatterino/chatterino2#2881] [Ideas] [answered]
Currently if you want to open someone's chat you have to manually create a new tab/split and input their username, or open their twitch channel from the usercard which is much slower and doesn't have chat history. It would be nice if there was a button in the usercard that opened that user's chat in a new tab/split :)

**Comment by Infinitay** [comment]
Closed via #3625.
Further built upon in #3766

**Comment by pajlada** [comment, marked as answer]
I consider this implemented in https://github.com/Chatterino/chatterino2/pull/3625
